apps/rush-lib/src/logic/taskRunner/ProjectTask.ts

```diff
@@ -71,6 +71,10 @@ export class ProjectTask implements ITaskDefinition {
     this._packageDepsFilename = options.packageDepsFilename;
   }
 
+  public static getTaskName(rushProject: RushConfigurationProject): string {
```

Comment:
Just curious, why is it useful to introduce a function that merely returns `.packageName`?  I'm not saying it's a bad idea, but it might be helpful to explain in a code comment

Reply:
Basically, there's some logic based off of using task names that was making the assumption that `.packageName` was the task name. To remove this assumption, I just created a static method to obtain it so that we know it should be the task name. I'll add a comment